src/PKSim.Presentation/Services/SimulationResultsTask.cs

```diff
@@ -66,7 +68,15 @@ private void cloneIndividualSimulationResults(IndividualSimulation sourceSimulat
          //Step 2 - update the data repository based on the results created above
          targetSimulation.DataRepository = _dataRepositoryCreator.CreateResultsFor(targetSimulation);
 
+
+         var curveNamesToRename = _curveNamer.CurvesWithOriginalName(sourceSimulation, sourceSimulation.Charts).Select(x => x.Name);
+
          sourceSimulation.SimulationCharts.Each(c => targetSimulation.AddAnalysis(_chartTemplatingTask.CloneChart(c, targetSimulation)));
+         targetSimulation.Charts.SelectMany(chart => chart.Curves).Where(curve => curveNamesToRename.Contains(curve.Name)).
+            Each(curve =>
+            {
+               curve.Name = _curveNamer.CurveNameForColumn(targetSimulation, curve.yData);
```

Comment:
This should probably be a method from the CurveName as well. This is exact duplicate as in RenameBuildingBlockTask